### PR TITLE
Pi 8297 Product images removed after returning from reviews page

### PR DESCRIPTION
**Pull request by CarinaHoffmann**

### CODE REVIEW

File: pages/Product/subscriptions.js [hunk 1]
```diff
@@ -59,7 +60,13 @@ export default function product(subscribe) {
    * Gets triggered on entering the product details route.
    */
   subscribe(productRouteDidEnter$, ({ dispatch, getState }) => {
-    dispatch(getProductData());
+    /*
```
Comment (richardgorman) [inline pages/Product/subscriptions.js]: Please use the usual comment style.

Comment (CarinaHoffmann) [reply pages/Product/subscriptions.js]: Done.

File: pages/Product/actions/getProductData.js [hunk 2]
```diff
@@ -33,11 +37,13 @@ const getProductData = (selectedVariantId = null, baseProductId = null) =>
 
     dispatch(requestProductData(productId, selectedVariantId));
 
-    if (!selectedVariantId) {
+    if (!selectedVariantId && (currentProductId !== parentId)) {
```
Comment (richardgorman) [inline pages/Product/actions/getProductData.js]: Please add a short comment here.

File: pages/Product/actions/getProductData.js [hunk 3]
```diff
@@ -33,11 +37,13 @@ const getProductData = (selectedVariantId = null, baseProductId = null) =>
...
       dispatch(setProductId(parentId));
     }
 
-    dispatch(setProductVariantId(selectedVariantId));
+    if (currentVariantId !== selectedVariantId) {
```
Comment (richardgorman) [inline pages/Product/actions/getProductData.js]: Please add a short comment here.

Comment (CarinaHoffmann) [reply pages/Product/actions/getProductData.js]: Done.

File: pages/Product/actions/getProductData.js (hunk 2 above)
Comment (CarinaHoffmann) [reply pages/Product/actions/getProductData.js]: Done.

**Pull request merged**